rest/src/main/java/com/netflix/conductor/rest/rbac/UserType.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2024 Netflix, Inc.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package com.netflix.conductor.rest.rbac;
+
+import java.util.List;
+
+public class UserType {
+
+    private List<String> groupsAndRoles;
```

Comment:
I would rename groupsAndRoles to just roles, because groupsAndRoles for me seems a Map, not List.

Reply:
All groupsAndRoles variables as well as getters renamed.